ruby/ror/.gitignore

```diff
@@ -0,0 +1,33 @@
+# See https://help.github.com/articles/ignoring-files for more about ignoring files.
+#
+# If you find yourself ignoring temporary files generated by your text editor
+# or operating system, you probably want to add a global ignore instead:
+#   git config --global core.excludesfile '~/.gitignore_global'
+
+# Ignore bundler config.
+/.bundle
+
+# Ignore all logfiles and tempfiles.
+/log/*
```

Comment:
Do we need these? Are they absolute paths?

Reply:
we checked, a leading slash in gitignore doesn't mean absolute path. 

As stated in the documentation: https://git-scm.com/docs/gitignore
"A leading slash matches the beginning of the pathname. For example, "/*.c" matches "cat-file.c" but not "mozilla-sha1/sha1.c"."